pyglider/seaexplorer.py

```diff
@@ -387,6 +387,10 @@ def raw_to_timeseries(indir, outdir, deploymentyaml, kind='raw',
                         val = np.datetime64("1970-01-01") + val_us
                     else:
                         val = np.interp(time_timebase.astype(float), time_var.astype(float), var_non_nan)
+
+                        # interpolate only over those gaps that are smaller than 'maxgap'                        
+                        tg_ind = utils.find_gaps(time_var.astype(float),time_timebase.astype(float),maxgap)
```

Comment:
I believe this should be de-intented by one level so that it also applies the max gap check to variables with the datetime dtype

Reply:
Good call. Done.